crc/models/study.py

```diff
@@ -165,18 +186,14 @@ def update_model(self, study_model: StudyModel):
         if status == StudyStatus.open_for_enrollment:
             study_model.enrollment_date = self.enrollment_date
 
-        # change = {
-        #     'status': ProtocolBuilderStatus(self.protocol_builder_status).value,
-        #     'comment': '' if not hasattr(self, 'comment') else self.comment,
-        #     'date': str(datetime.datetime.now())
-        # }
-
-        # if study_model.changes_history:
-        #     changes_history = json.loads(study_model.changes_history)
-        #     changes_history.append(change)
-        # else:
-        #     changes_history = [change]
-        # study_model.changes_history = json.dumps(changes_history)
+        study_event = StudyEvent(
+            study=study_model,
+            status=status,
```

Comment:
The study status is being saved here in the study event, but doesn't the status of the study itself need to change as well?

Reply:
It is being set on line 184